modules/tinymce/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed the dev ZIP missing the required `bin` scripts to build from the source #TINY-8542
 - Fixed a regression whereby text patterns couldn't be updated at runtime #TINY-8540
 
+### Improved
+- Naked buttons better adapt to various background colors, improved text contrast in notifications #TINY-8533
```

Comment:
```suggestion
- Fixed a regression whereby text patterns couldn't be updated at runtime #TINY-8540
- Naked buttons better adapt to various background colors, improved text contrast in notifications #TINY-8533
```
Improvements can't go in a patch release and the Jira is a bug, so this should be under the Fixed section

Reply:
Merging this suggestion so that we can get this merged today.